twoliter/src/cmd/mod.rs

```diff
@@ -35,6 +37,9 @@ pub(crate) enum Subcommand {
 
     Make(Make),
 
+    /// Update Twoliter.lock
+    Update(Update),
```

Comment:
@bcressey do we want to use `twoliter update` for this purpose? Are there other things we might update in the future? If so should we have this be `twoliter update lock`?